packages/dev/core/src/Audio/sound.ts

```diff
@@ -929,8 +932,9 @@ export class Sound {
                 }
                 this.isPlaying = false;
                 this.isPaused = true;
-            } else if (Engine.audioEngine?.audioContext) {
-                this._stopSoundSource();
+            } else if (Engine.audioEngine?.audioContext && this._soundSource) {
+                this._soundSource.onended = () => void 0;
+                this._soundSource.stop();

```

Comment:
Same remark, It also means we do not disconnect the source anymore ? (just wondering if intended)

Reply:
I am wondering if disconnect is even needed?